icicle/CMakeLists.txt

```diff
@@ -108,6 +108,11 @@ if (HASH)
   setup_hash_target()
 endif()
 
+option(BARRET "Use Barret representation" OFF)
```

Comment:
This won't be exposed to Rust and Go as a feature right? @mickeyasa @LeonHibnik

Reply:
Right, this shouldn't be exposed

Reply:
Why is this a compile time decision anyway?
Are we planning on distributing two binaries, one for barret and another for mont?

Reply:
No. It was an option but complexity is not worth it